test/GovernerAlpha.test.js

```diff
@@ -0,0 +1,350 @@
+const BTRUST = artifacts.require("BTRUST");
```

Comment:
do you think we should split this up into multiple files like they do in the comp repo? as in https://github.com/compound-finance/compound-protocol/tree/master/tests/Governance/GovernorAlpha ?

Reply:
not needed imo. better we have them in one place to avoid confusion later for us or future developers.